calm_adapter/terraform/deletion_checker_worker.tf

```
@@ -0,0 +1,58 @@
+module "calm_deletions_checker_queue" {
+  source                     = "git::github.com/wellcomecollection/terraform-aws-sqs//queue?ref=v1.1.2"
+  queue_name                 = "calm-deletions-checker"
```

minor:

```suggestion
  queue_name                 = "calm-deletions-checker-input"
```

I know it being an input is implicit from being a queue, but that's not always the case, and I find it useful to have them distinguished when I'm looking through the console.